### ISSUE TITLE
Fatigue in Vashjir

### ISSUE THREAD

**Issue by Ibuprofen5961** [Valhalla-Gaming/Tracker#4422]
Theres Fatigue everywhere in vashjir i cant even go there without dying 10 times

**Comment by Trekex** [comment]
Yes there is a way to get there... stay low on the sea bed and in many points you won't get fatigue :)

**Comment by Ibuprofen5961** [comment]
there are some places u cant go too because not all vashjir is low
